test/integration/test_integration_xarray_plugin.py

```diff
@@ -0,0 +1,25 @@
+import os.path
+
+import pytest
+
+from test.conftest import TEST_INPUT_DATA_DIR
+
+xr = pytest.importorskip("xarray", minversion="0.17.1.dev0")
```

Comment:
Does this work with `0.17.0`?

Reply:
No, it will work with the next release of *xarray*. At the moment the test runs only on `master`.

Reply:
Okay. Mind adding a comment to update `minversion="0.18"` after the next release of `xarray`?

Reply:
Looks like it should work with 0.17: https://github.com/pydata/xarray/blob/v0.17.0/xarray/backends/common.py#L346

Reply:
Bits and pieces of the implementation where already present in 0.17.0, but the switch to the new plugin API got in after the release pydata/xarray#4989 (I know as I did the PR).

Reply:
That is good to know, thanks for the clarification :+1: